users/api/http/responses.go

```diff
@@ -50,3 +56,20 @@ func (res identityRes) Headers() map[string]string {
 func (res identityRes) Empty() bool {
 	return false
 }
+
+type resetPassRes struct {
+	Msg   string `json:"msg"`
+	Error string `json:"error"`
```

Comment:
I'm not sure if you should have an error here. @blokovi is working on the PR that should tackle error handling and HTTP error response.

Reply:
I'm not sure either, but something needs to be displayed in frontend. For example if token is expired. We can remove this error. Maybe that "token expired", and "non-existent" user messages can be in Msg field.